wideq/client.py

```diff
@@ -415,41 +451,94 @@ class Device(object):
     def __init__(self, client: Client, device: DeviceInfo):
         """Create a wrapper for a `DeviceInfo` object associated with a
         `Client`.
-        """
+        """        
         self.client = client
         self.device = device
         self.model: ModelInfo = client.model_info(device)
 
+    def _set_operation(self, value):
+        """Set a device's operation for a given `value`.
+        """
+        
+        self.client.session.set_device_controls(
+            self.device.id,
+            value,
+            )
```

Comment:
This method doesn't seem to be called anywhere. Can you clarify what it's useful for?